<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>77</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="gbox_options_tab1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>191</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_filter">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="in_filter_class">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>80</y>
             <width>311</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19]</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="gbox_filter">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>251</width>
             <height>91</height>
            </rect>
           </property>
           <property name="title">
            <string>Filter type</string>
           </property>
           <widget class="QRadioButton" name="radio_btn_filter_only">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>211</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Only these objects and others</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="radio_btn_filter_include">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>221</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Include these objects and othres</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QPushButton" name="btn_gen_text">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>250</y>
             <width>121</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Genrate text file</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_num_objects">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>120</y>
             <width>261</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Number of Objects:</string>
           </property>
          </widget>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>350</y>
             <width>521</width>
             <height>281</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="in_text_format">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>320</y>
             <width>551</width>
             <height>22</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>{0:02} {1:03} {2:03} {3:03} {4:03} {5:03} {6:03} {7:03} {8:03} {9} {10} {11} {12}</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_format">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>290</y>
             <width>611</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>6</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> [0]class , [1]y_Max  [2]x_Max [3]y_Min [4]x_Min [5]H [6]W[7] x_Center [8]y_Center [9]Diff [10]Name [11-12] size</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_create_dataloader_torch">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>650</y>
             <width>221</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Sava it as Pytorch data Loader</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chck_box_scale_to_1">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>240</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Scale to 1</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
